KEYS

```diff
@@ -0,0 +1,4824 @@
+Edit this file in SVN at https://svn.apache.org/repos/asf/maven/project/KEYS
+
+Then publish to download area /dist/maven/KEYS by
+
+svn co https://dist.apache.org/repos/dist/release/maven dist-maven
+cd dist-maven
+svn cat https://svn.apache.org/repos/asf/maven/project/KEYS > KEYS
+
+then commit.
+Download area publication is a PMC activity. If you are not a PMC member
+then email private@maven.apache.org with a request to publish your
+modification.
```

Comment:
Should we update this instruction also in this PR?

Reply:
this PR was about all the svn2git work to get the history (as stated in title)
but of course, updating instructions will be the next step (and I won't do a PR: this is a proof of CTR efficiency for basic updates)

Reply:
even with CRT and simple changes PR can be valuable 😄

Reply:
yes, I fully understand and respect (no fight against)

but TBH, when I am working on 10's of different aspects in parallel in many different Git repos (at ASF and others), it costs so much energy to open PRs here and there, wait, come back on PRs that I remember (eventually forget some), check
=> doing quickly a first commit then having other's feedback by improvement commits on top of the initial commit: this way of reviewing puts less stress and energy cost to me (and review comes in the form of improvements, which I find more positive)

on this one, I wanted a PR because I wanted RTC, but I see that I missed that I need to explain a little bit more my plan as it was not fully obvious that the plan is to migrate one step further from svn to Git

Sorry